Gems/ONNX/Code/Source/Clients/Mnist.cpp

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "Mnist.h"
+
+namespace ONNX
+{
+    template<typename T>
+    static void MNIST::softmax(T& input)
```

Comment:
Actually the namespace that should be open here is `MNIST` and not ONNX.

Also per the Coding Standards Abbreviations must be the same case as regular words: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#abbreviations

Reply:
I've placed Mnist in its own namespace and changed the naming to comply with the coding standards.

Reply:
Does this also mean that the name of the gem should be changed from ONNX to Onnx, along with the namespace and all the generated parts like ONNXSystemComponent be changed to OnnxSystemComponent etc?